tests/Feature/Http/Livewire/StreamListArchiveTest.php

```diff
@@ -26,11 +27,16 @@
 
 it('shows streamers as dropdown options', function() {
     // Arrange
-    Channel::factory()
+    $channel_a = Channel::factory()
```

Comment:
Can you please combine the creation of the channel and the stream like: 

```
Channel::factory()
    ->for(Stream::factory()...
```

This way we have less code and see that the stream belongs to the channel.

Reply:
Not a problem, I'll sort that this week 🙂

Reply:
Sorted that @christophrumpel but let me know if it needs some more tweaks